mlflow/server/js/src/components/CompareRunView.js

```diff
@@ -42,15 +42,19 @@ class CompareRunView extends Component {
         <div className="run-metadata-container">
           <div className="run-metadata-label">Run UUID:</div>
           <div className="run-metadata-row">
-            {this.props.runInfos.map((r) => <div className="run-metadata-item">{r.getRunUuid()}</div>)}
+            {this.props.runInfos.map((r, idx) =>
+              <div className="run-metadata-item" key={idx}>
+                {r.getRunUuid()}
+              </div>
+            )}
           </div>
         </div>
         <div className="run-metadata-container last-run-metadata-container">
           <div className="run-metadata-label">Start Time:</div>
           <div className="run-metadata-row">
-            {this.props.runInfos.map((run) => {
+            {this.props.runInfos.map((run, idx) => {
                const startTime = run.getStartTime() ? Utils.formatTimestamp(run.getStartTime()) : '(unknown)';
-               return <div className="run-metadata-item">{startTime}</div>;
+               return <div className="run-metadata-item" key={idx}>{startTime}</div>;
```

Comment:
Maybe the key should be the run ID, not index, in case we let people add runs to this view dynamically later. Seems unlikely but it's better if the key is obviously unique. Same thing applies in the map() above.